Files/Views/SettingsPages/Multitasking.xaml

```diff
@@ -3,16 +3,612 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Files.UserControls.Settings"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d">
+    <Page.Resources>
+        <Style x:Key="RightAlignedToggleSwitchStyle" TargetType="ToggleSwitch">
```

Comment:
What do you think about moving this style to it's own file so that it isn't defined in all the settings pages.

Reply:
Good idea, I'll fix this.